Lib/test/test_csv.py

```diff
@@ -1083,10 +1115,15 @@ class mydialect(csv.Dialect):
                          '"lineterminator" must be a string')
 
     def test_invalid_chars(self):
-        def create_invalid(field_name, value):
+        def create_invalid(field_name, value, **kwargs):
             class mydialect(csv.Dialect):
-                pass
+                delimiter = ','
+                quoting = csv.QUOTE_ALL
+                quotechar = '"'
+                lineterminator = '\r\n'
```

Comment:
This test was broken. In most cases the exception was raised for different reason, not because wrong specified value, but because of the lack of required default values.